lib/mix/tasks/ecto.rollback.ex

```diff
@@ -117,6 +124,9 @@ defmodule Mix.Tasks.Ecto.Rollback do
         do: Keyword.merge(opts, log: false, log_migrations_sql: false, log_migrator_sql: false),
         else: opts
 
+    log_level = String.to_existing_atom(Keyword.get(opts, :log_level, "debug"))
```

Comment:
I think we should do it conditionally, as above, in case the user does call it after their application is started.